Basic Join/Average Population of Each Continent.sql

```diff
@@ -1 +1 @@
-select country.continent, floor(avg(city.population)) from country join city on city.countrycode = country.code group by country.continent;
\ No newline at end of file
+select country.continent, floor(avg(city.population)) from country join city on city.countrycode = country.code group by country.continent;
```

Comment:
Could you please delete this file and pull it again since there's no change in the file. This way it will not show a change in the PR. Thanks!